lnwallet/rpcwallet/remote_signer_builder.go

```diff
@@ -66,3 +65,17 @@ func (b *RemoteSignerBuilder) createInboundRemoteSigner() (
 		b.cfg.Timeout,
 	)
 }
+
+// createOutboundRemoteSigner creates a new OutboundRemoteSigner instance.
+// The function returns the created OutboundRemoteSigner instance, and a cleanup
+// function that should be called when the OutboundRemoteSigner is no longer
+// needed.
```

Comment:
why the extra method which only calls one thing?

Reply:
Thanks! Removed :)